Build of PR #4311 (git tag 9955e8e) fails

## Expected Behavior

Build succeeds.

## Actual Behavior

Build fails.

```
/tmp/hello_hpx/Debug-default/build/_deps/hpx-src/libs/version/src/version.cpp:11:10: fatal error: hpx/config/config_strings.hpp: No such file or directory
 #include <hpx/config/config_strings.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                        
compilation terminated.
```

## Steps to Reproduce the Problem

  1. `git clone https://github.com/kordejong/hello_hpx.git`
  1. `cd hello_hpx`
  1. `bash ./debug_build.sh`

## Specifications

... Please describe your environment

  - HPX Version: 1.4.1 + APEX fixes
  - Platform (compiler, OS): gcc-7, Kubuntu 18.04

--- a/libs/version/CMakeLists.txt
+++ b/libs/version/CMakeLists.txt
@@ -31,4 +31,4 @@ add_hpx_module(version
   CMAKE_SUBDIRS
 )
 
-target_include_directories(hpx_version PRIVATE $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}>)
+target_include_directories(hpx_version PRIVATE $<BUILD_INTERFACE:${HPX_BINARY_DIR}>)

